public/js/logix.js

```diff
@@ -1481,6 +1482,26 @@ CircuitElement.prototype.layoutProperties = {
     downDimensionY: 5
 };
 
+// Default mutable properties in layout mode 
+// Should we define it for subcircuit elements only??
+CircuitElement.prototype.subcircuitMutableProperties = {
+    "label": {
+        name: "Label: ",
+        type: "text",
+        func: "setLabel"
+    },
```

Comment:
No the label has to be common with the normal mode. The label direction however should be different. This makes sense as the circuit design is different from the subcircuit layout design but labels will not change.

Reply:
As the function `setLabel` is the same. Whenever we change the label in either mode, it changes the original label. I can remove it from `CircuitElement.prototype.subcircuitMutableProperties` and this will remove it from showing in layout mode.

Reply:
Yes please remove it.

Reply:
removed